app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -30,7 +42,11 @@ KT.hosts.fetchContentViews = function () {
             select.find('option').remove();
             select.append($("<option />"));
             $.each(data.results, function(index, view) {
-                select.append($("<option />").val(view.id).text(view.name));
+                option = $("<option />").val(view.id).text(view.name);
```

Comment:
I'm not sure this is entirely necessary but I was surprised that the code wasn't attempting to select the previously chosen CV when the list was reloaded